content/en/docs/setup/upgrade/in-place/index.md

```diff
@@ -12,6 +12,10 @@ the upgrade, it checks that the Istio installation meets the upgrade eligibility
 criteria. Also, it alerts the user if it detects any changes in the profile
 default values between Istio versions.
 
+{{< tip >}}
+[Canary Upgrade](/docs/setup/upgrade/canary/) is much safer than doing an in-place upgrade and is the highly recommended upgrade method.
```

Comment:
My concern here is the `much` and `highly`. Is it better to say `Canary Upgrade is safer than doing an in-place upgrade and is the recommended upgrade method.`?

Reply:
Sounds better